workers/unity/Assets/Gdk/Core/Worker/WorkerBase.cs

```diff
@@ -19,18 +20,23 @@ public abstract class WorkerBase : IDisposable
 
         public abstract string GetWorkerType { get; }
 
-        protected WorkerBase(string workerId, Vector3 origin)
+        protected WorkerBase(string workerId, Vector3 origin, Option<IErrorDispatcher> errorDispatcher)
```

Comment:
This `Option<>` basically serves as a default argument? Is there any reason you can't do: 

`protected WorkerBase(string workerId, Vector3 origin, IErrorDispatcher errorDispatcher = new LoggingDispatcher())`